testcontainers/tests/dual_stack_host_ports.rs

```diff
@@ -0,0 +1,73 @@
+use std::net::{Ipv6Addr, TcpListener};
+
+use testcontainers::{clients, core::WaitFor, images::generic::GenericImage};
+
+/// Test the functionality of exposing container ports over both IPv4 and IPv6.
+#[tokio::test]
+async fn test_ipv4_ipv6_host_ports() {
+    let _ = pretty_env_logger::try_init();
+    let docker = clients::Cli::default();
+
+    let wait_for = WaitFor::message_on_stdout("server is ready");
+    let image =
+        GenericImage::new("tumdum/simple_web_server", "latest").with_wait_for(wait_for.clone());
```

Comment:
Where is this coming from? I would like to either depend on images that are known to be maintained or if you need a specific one for your usecase, you can add one in the `testimages` crate!

Reply:
I copied this out of the `generic_image_with_custom_entrypoint` test. I can take a crack at re-implementing it in `testimages`.

Reply:
Right, I do remember this now as well 😅

I would much appreciate if we could move this image into `testimages` 😊